en/docs/develop-components/work-with-the-micro-integrator-runtime-in-choreo.md

```diff
@@ -61,10 +61,83 @@ Here are some of the benefits of adding third-party JARs to the Micro Integrator
  - Optimized performance: By offering optimized implementations of common functions, third-party libraries can improve the performance of the Micro Integrator.
  - Efficient development: Developers can avoid re-implementing commonly used functions, reducing development time and increasing efficiency.
 
-To incorporate third-party libraries into your Micro Integrator project, create a new directory called `libs` at the root of your project. If your project root and GitHub repository root are not the same, place the `libs` directory in the project root. After adding the required JAR files to the `libs` directory, the Micro Integrator runtime will automatically include them when deploying the component. This process ensures that any dependencies needed by your integration flow are available at runtime.
+To incorporate third-party libraries into your Micro Integrator project, create a new directory called `libs` at the
+root of your project. If your project root and GitHub repository root are not the same, place the `libs` directory in
+the project root. After adding the required JAR files to the `libs` directory, the Micro Integrator runtime will
+automatically include them when deploying the component. This process ensures that any dependencies needed by your
+integration flow are available at runtime.
+
+In addition to `jar` libraries, you may also need to incorporate OSGi bundle JARs into your project. To do this,
+create another directory called `dropins` at the root of your project. Similarly, if the project root and GitHub
+repository root differ, place the `dropins` directory in the project root.
+
+Within the `dropins` directory, add the OSGi bundle JAR files. These bundles will be picked up by the OSGi runtime during
+deployment, enabling you to utilize their functionality within your Micro Integrator project.
 
 ![Libs directory](../assets/img/develop-components/micro-integrator/libs_dir.png)
 
+## Importing custom certificates to Micro Integrator
+
+The feature allows Choreo MI users to import certificates to the MI `client-truststore.jks` file. This is useful when
+users need to connect to servers or services that use SSL/TLS encryption and have self-signed or certificates issued by private CAs
+that are not trusted by default. By importing the certificate to the MI client-truststore.jks file, MI clients can
+establish a secure connection with the server or service without encountering SSL/TLS errors or warnings.
+
+Before importing the certificate, you need to obtain the certificate file. This file can be in
+the PEM or DER format. You can obtain the certificate file from the server or service provider or by exporting it from a
+web browser. Certificates should be added to the `/wso2mi/certs/` folder. Therefore, users must ensure that the mount path
+for the certificate file follows this path
+
+1. Go to the Component page and click on the **Deploy** page for the component you want to add the certificate to.
```

Comment:
1. In the Choreo console, select the component you wish to add a certificate to. 
2. From the left navigation click **Deploy**.